docs/README.md

```diff
@@ -1,6 +1,7 @@
 # Docs
 
 The docs are built using [Docusaurus 2](https://v2.docusaurus.io/).
+Nodejs version 12 is used to run Docusaurus.
```

Comment:
Let's use active voice and deploy their official naming convention. Do we know which version exactly we're using?
```suggestion
To run Docusaurus, install `Node.js` version 12.
```

Reply:
We should also add this update in this section: https://github.com/RasaHQ/rasa#deploying-documentation-updates

Reply:
> Do we know which version exactly we're using?
The CI is set up to use `12.x`